### PR TITLE
Add n3ds skater v3077 rop addr

### PR DESCRIPTION
**Pull request by d3m3vilurr**
Not find APPHEAP_PHYSADDR yet.

This rop addr fully work except download ropbin.
(I don't know why, it stuck gray screen step, can't download my site)

So, ropbin renamed and copy to sdcard, it can launch hblauncher.

### CODE REVIEW

**Comment by yellows8** [comment]
"copy to sdcard, it can launch hblauncher." Booting *hax payload actually works fine?(*hax payload doesn't actually support KOR AFAIK...)

**Comment by d3m3vilurr** [comment]
yeah. i pushed new pr to ninjhax 2.x repo :)

**Comment by d3m3vilurr** [comment]
https://github.com/smealum/ninjhax2.x/pull/7 :D
https://twitter.com/d3m3vilurr/status/723549707193049088

File: 3dsbrowserhax_rop_skater_kor_v3077.php [hunk 1]
```diff
@@ -0,0 +1,84 @@
+<?php
+$CODEBLK_ENDADR = ((0x00100000 + 0x00277000 + 0x00065000 + 0x0000A000 + 0x00045920) + 0xfff) & ~0xfff;
+
+$OSSCRO_HEAPADR = 0x08133000;
+$WEBKITCRO_HEAPADR = 0x083ff000;
+$PEERCRO_HEAPADR = 0x0831a000;
+$APPHEAP_PHYSADDR = 0x2b000000; // unknown
```
Comment (yellows8) [inline 3dsbrowserhax_rop_skater_kor_v3077.php]: This "//unknown" comment should be removed since the hax works fine for you.

**Comment by d3m3vilurr** [comment]
done

**Comment by yellows8** [comment]
I git-pulled on the server hosting browserhax, can you test with KOR there?

**Comment by d3m3vilurr** [comment]
hm... crash after yellow screen

**Comment by d3m3vilurr** [comment]
data looks same my [site](http://bhx.suod.kr/browserhax_fright.php). my site work, but orig site still crash after yellow. i don't know that.. :(

**Comment by d3m3vilurr** [comment]
@yellows8 update. 
- browserhax_fright.php : crash after yellow
- browserhax_fright_tx3g.php : work

**Comment by yellows8** [comment]
Are you sure you only have one tab open, and how exactly did you test it?

**Comment by d3m3vilurr** [comment]
Hum... I was not use single tab. and i found my mistake.
so this is retesting step:
1. close all browser tabs, and close browser
2. L+R in home, and read qr tag then open browser
3. view update popup
4. reboot device
5. open browser

now all php works. :)

**Comment by yellows8** [comment]
"now all php works" :)

**Pull request merged**